tests/saitests/py3/sai_qos_tests.py

```diff
@@ -5719,14 +5720,14 @@ def runTest(self):
 
         try:
             for i in range(len(prio_list)):
-                log_message("DSCP index {}/{}".format(i + 1, len(prio_list)))
+                print("DSCP index {}/{}".format(i + 1, len(prio_list)), file=sys.stderr)
```

Comment:
what's the reason change log to print?

Reply:
I needed the stderr printing for it to show up. 
Noticed I can use log_message with the to_stderr option, went ahead and revised to use that.